charts/newrelic-k8s-metrics-adapter/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 description: A Helm chart to deploy the New Relic Kubernetes Metrics Adapter.
 name: newrelic-k8s-metrics-adapter
 version: 1.2.0
-appVersion: 0.4.2
```

Comment:
Revert this appVersion bump. We do a single appVersion bump during the release process.